frontend/public/components/build.tsx

```diff
@@ -61,11 +61,27 @@ export const BuildsDetails: React.SFC<BuildsDetailsProps> = ({obj: build}) => {
   </div>;
 };
 
-const envPath = ['spec', 'strategy', 'sourceStrategy'];
+export const getStrategyType = (strategy) => {
```

Comment:
Doesn't need to be exported.

Reply:
@spadgett these need to be exported because their also used in build-config

Reply:
Yeah, I see now. Sorry, missed that.